### PR TITLE
Add test to validate datasource name/url

### PR DESCRIPTION
**Pull request by shruacha1234**
Signed-off-by: Shruthi <Shruthi.Acharya@ibm.com>

### CODE REVIEW

File: tests/autotune_test_yamls/manifests/da/app_autotune_yaml/datasource_name/valid-datasource-name.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  mode: "show"
+  selector:
+    matchLabel: "app.kubernetes.io/name"
+    matchLabelValue: "petclinic-deployment"
+    matchRoute: ""
+    matchURI: ""
+    matchService: "https"
+  datasource:
+    name: "prometheus"
+    value: "http://192.168.49.2:31609"
```
Comment (chandrams) [inline tests/autotune_test_yamls/manifests/da/app_autotune_yaml/datasource_name/valid-datasource-name.yaml:24]: IP and Port no. should not be hardcoded. This should be replaced dynamically when the test is run.

File: tests/autotune_test_yamls/manifests/da/app_autotune_yaml/datasource_url/blank-datasource-url.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      value_type: "double"
+  mode: "show"
+  selector:
+    matchLabel: "app.kubernetes.io/name"
+    matchLabelValue: "petclinic-deployment"
+    matchRoute: ""
+    matchURI: ""
+    matchService: "https"
+  datasource:
+    name: "prometheus"
```
Comment (chandrams) [inline tests/autotune_test_yamls/manifests/da/app_autotune_yaml/datasource_url/blank-datasource-url.yaml:23]: You should not be using "prometheus" for all datasource  tests, change it to prometheus1 or something. Also add a test with multiple datasources.

**Review by chandrams — APPROVED** [review]
LGTM

**Review by dinogun — APPROVED** [review]
lgtm

**Comment by dinogun** [comment]
Please fix the conflicts

**Pull request merged**